pkg/apis/autoscaling/register.go

```diff
@@ -33,4 +33,6 @@ const (
 	// KPALabelKey is the label key attached to a K8s Service to hint to the KPA
 	// which services/endpoints should trigger reconciles.
 	KPALabelKey = GroupName + "/kpa"
+
+	// TODO: plumb through HPA CPU target annotation.
```

Comment:
Without this being plumbed through, is there a sensible default set?

Reply:
The default comes from the HPA, which is 75%.